static/app/components/events/autofix/index.tsx

```diff
@@ -0,0 +1,27 @@
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import {AutofixBanner} from 'sentry/components/events/autofix/autofixBanner';
+import {AutofixCard} from 'sentry/components/events/autofix/autofixCard';
+import type {GroupWithAutofix} from 'sentry/components/events/autofix/types';
+import {useAiAutofix} from 'sentry/components/events/autofix/useAutofix';
+import type {Event} from 'sentry/types';
+
+interface Props {
+  event: Event;
+  group: GroupWithAutofix;
+}
+
+export function Autofix({event, group}: Props) {
+  const {autofixData, triggerAutofix, reset} = useAiAutofix(group, event);
+
+  return (
+    <ErrorBoundary mini>
```

Comment:
great idea, so our experimental work doesn't mess up the critical issues page lol